test/tstRay.cpp

```diff
@@ -347,19 +355,25 @@ BOOST_AUTO_TEST_CASE(overlap_distance_sphere,
   {                                                                            \
     float t0;                                                                  \
     float t1;                                                                  \
-    constexpr auto inf = KokkosExt::ArithmeticTraits::infinity<float>::value;  \
     BOOST_TEST(!ArborX::Experimental::intersection(ray, sphere, t0, t1));      \
     BOOST_TEST((t0 == inf && t1 == -inf));                                     \
   } while (false)
 
 BOOST_AUTO_TEST_CASE(ray_sphere_intersection,
                      *boost::unit_test::tolerance(1e-6f))
 {
+  #ifdef _MSC_VER
+  auto const inf = KokkosExt::ArithmeticTraits::infinity<float>::value;
+#else
+  constexpr auto inf = KokkosExt::ArithmeticTraits::infinity<float>::value;
+#endif
   using ArborX::Sphere;
   using ArborX::Experimental::Ray;
-
+  #ifdef _MSVC_VER
```

Comment:
`_MSVC_VER`is never defined yet the code still works. This means this change is useless.